cli/src/commands/enrich/mod.rs

```diff
@@ -0,0 +1,89 @@
+use crate::commands::enrich::dependency_track::DependencyTrackArgs;
+use crate::commands::enrich::epss::EpssProvider;
+use crate::commands::enrich::snyk::{SnykArgs, SnykProvider};
+use crate::Error;
+use clap::builder::PossibleValue;
+use clap::{Parser, ValueEnum};
+use std::str::FromStr;
+
+/// Dependency Track enrichment command handler.
+pub mod dependency_track;
+/// EPSS enrichment command handler.
+pub mod epss;
+/// Snyk enrichment command handler.
+pub mod snyk;
+
+/// Enumerates the supported enrichment providers.
+#[derive(Clone, Debug)]
+pub enum EnrichmentProviderKind {
+    /// Use the Dependency Track enrichment provider.
+    DependencyTrack,
+    /// Use the EPSS enrichment provider.
+    Epss,
+    /// Use the Snyk enrichment provider.
+    Snyk,
+}
+
+/// The Enrich Command handler.
+pub async fn execute(args: &EnrichArgs) -> Result<(), Error> {
+    match args.provider {
+        EnrichmentProviderKind::DependencyTrack => {
```

Comment:
No!

Reply:
As it stands, I'm not passing this if the DT stuff is in it.  We have to talk about this.